full/Dockerfile

```diff
@@ -204,41 +204,41 @@ ENV PATH=$PATH:/home/gitpod/.nvm/versions/node/v${NODE_VERSION}/bin
 LABEL dazzle/layer=lang-python
 LABEL dazzle/test=tests/lang-python.yaml
 USER gitpod
-ENV PATH=$HOME/.pyenv/bin:$HOME/.pyenv/shims:$PATH
-RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash \
```

Comment:
I don't think we should remove pyenv, it makes things easier to gitpodify and removing might break a *lot* of builds.

Reply:
pyenv basically breaks a ton of tools, such as poetry, and doesn't work well in my opinion

Reply:
How do you suggest people who do not know docker manage python versions in gitpod?

Reply:
Using python as the base docker image in the gp yml?

I'm going to add back Pyenv, but am still going to install the main python version like this.

The reason this is better is that it literally overrides the system preinstalled version of python, as well as fixes a lot of issues.

Reply:
Good plan!